README.rst

```diff
@@ -20,6 +20,20 @@ current) stage3 Docker container.
    :alt: 'Waffle.io - Columns and their card count'
 
 
+Requirements
+------------
+
+You require `Docker <https://wiki.gentoo.org/wiki/Docker>`_ and `FUSE
+<https://wiki.gentoo.org/wiki/Filesystem_in_Userspace>`_. Docker must be
+configured to use the devicemapper storage driver.  This can be achieved with
+the following inside ``etc/docker/daemon.json``:
```

Comment:
Could you add this link: https://docs.docker.com/storage/storagedriver/device-mapper-driver/ ? Could you also add a `/` so that it's `/etc/docker...`?

Reply:
Done!